services/frontend/src/scenes/Profile/ProfileLayout.js

```diff
@@ -28,7 +28,7 @@ const sections = [
     }
 ];
 
-const ProfileLayout = ({isLoading, isUserProfile, profile, projects, skills}) => (
+const ProfileLayout = ({databaseSkills: databaseSkills, isLoading, isUserProfile, profile, projects, skills}) => (
```

Comment:
Is there any reason this is `databaseSkills: databaseSkills` instead of just `databaseSkills`?

Reply:
I think it got added when I was replacing